shared/utils/il2cpp-functions.cpp

```diff
@@ -564,6 +559,6 @@ void il2cpp_functions::Init() {
 
     dlclose(imagehandle);
     initialized = true;
+    usleep(5000L);  // 0.005s
```

Comment:
Do we need to _always_ delay execution in `il2cpp_functions::Init`? It seems like this could be avoided. Why was this required?

Reply:
These pauses are to allow the logging to finish. I still don't know exactly when/why it happens, but without these pauses (and sometimes with them) some logging messages just get clobbered out of existence.
I've read that there's some sort of ring buffer, but my current hypothesis is that once the buffer is full, subsequent logs just won't work. So after we log lots, I sleep.

These pauses are honestly quite minuscule though.

Reply:
The first pause is for all the dlsym logs.
The 2nd pause is for the logs generated by all of the Instruction parsing that xref tracing does.